osimpipeline/study.py

```diff
@@ -188,8 +196,22 @@ def add_task(self, cls, *args, **kwargs):
     def add_task_cycles(self, cls, *args, **kwargs):
```

Comment:
Could you show some example code that uses this with setup tasks as a kwarg (either in the GitHub comments or in the docstring)?